### ISSUE TITLE
Bug my tab in Python code

### ISSUE THREAD

**Issue by AlanAlbuquerque** [SublimeCodeIntel/SublimeCodeIntel#475]
Firstly, I love this Package for the SublimeText, congratulations for all developers than created this.

If I'm typing and press TAB, I need TAB command to indent my code, but if I type, for example:

<code>
raise Exception as e:
</code>

after I write the character ':' , the SublimeCodeIntel showed me the suggestion 'AttributeError',  if I press TAB will write 'AttributeError', but I want the TAB command. Only indents if I press TAB again.

This is happening with others situations, if already exists a line and insert my cursor in start of line and press TAB, on first TAB will write the suggestion, on second TAB will erase the suggestion and write my indents.

In other words, I need press two times the TAB for create one indent block

I need some setting for delay the suggestion or only select suggestions if pressing ENTER.

**Comment by spectacles** [comment]
Remove SublimeCodeIntel and install CodeComplice via packageControl. It's a dork of this plugin, you won't have this problem there!

**Comment by spectacles** [comment]
Actually it's a fork not a dork. Anyways!

**Comment by AlanAlbuquerque** [comment]
@spectacles thanks for the tip , but I have the same problem with CodeComplice , something that annoys is press ENTER and the suggestions appear, and I type automatically TAB and select the first suggestion without my will

**Comment by spectacles** [comment]
That's a slightly different problem, but non the less annoying, I agree. It is fixed in the upcoming version of CodeComplice, will be out tonight. Thank you for your feedback, sir!

**Comment by AlanAlbuquerque** [comment]
wow! it's fast! I than thank you for the attention.
